src/main/java/org/jabref/gui/openoffice/ManageCitationsDialogView.java

```diff
@@ -84,9 +84,6 @@ private Node getText(String citationContext) {
     }
 
     public boolean isOkToShowThisDialog() {
-        if (viewModel == null || viewModel.failedToGetCitationEntries) {
-            return false;
-        }
-        return true;
+        return (viewModel != null) && (!viewModel.failedToGetCitationEntries);
```

Comment:
at least the brace after the && are uncessary

Reply:
Let's see what checkstyle says... I think, it demands some braces?

Reply:
I think only in if conditions